src/templates/default/reviews/_includes/proposal_table.html

```diff
@@ -33,6 +33,21 @@
 				<i class="fa fa-sort-desc fa-lg"></i>
 			</a>
 		</th>
+		<th width="1%" class="hidden-md hidden-sm hidden-xs">
+			<select onchange="javascript:location.href = this.value;">
+				{% if not category %}
+				<option value="?order=count">ALL</option>
+				{% else %}
+				<option value="?order=count&category={{ category }}">{{ category }}</option>
+				<option value="?order=count">ALL</option>
+				{% endif %}
+				{% for category_option in category_options %}
+				{% if category != category_option %}
+				<option value="?order=count&category={{ category_option }}">{{ category_option }}</option>
```

Comment:
**issue:** the value `order=count` here will cause the proposals ordered only by the count after selecting the category even the user has chosen to ordered by other field like `lang`, `count`, etc. I guess we need to pass the query parameter from URL.

Reply:
Resolved in daa9f1bfcba8ff46b949f461027138e7a37e664d
use a trick to retain both order={}&category={} query params.